src/app/plugins/checkclasses.plugin.ts

```diff
@@ -15,10 +15,17 @@ export default class CheckClassesPlugin extends Plugin {
 
   private _MAX_CHANS_SHOWN: number = 10;
 
+  public override validate(message: IMessage, args: string[]) {
+    const valid = Moderation.Helpers.validateUser(args[0]);
+    if (!valid) {
+      return false;
+    }
+    return true;
```

Comment:
If `args.length` == 0 then this will crash, need to check for that

Also, just `return Moderation.Helpers.validateUser(args[0]);`

Reply:
bump

If there are no args, calling `args[0]` will crash